lib/Minify/Source/Factory.php

```diff
@@ -140,7 +159,7 @@ public function makeSource($spec)
 
         if ($this->options['checkAllowDirs']) {
             foreach ((array)$this->options['allowDirs'] as $allowDir) {
-                if (strpos($spec['filepath'], $allowDir) !== 0) {
```

Comment:
Hmm, the existing logic demands that the file be in *all* allowDirs, not any. Big bug!